tests/app/tests/test_webpack.py

```diff
@@ -67,6 +72,18 @@ def test_simple_and_css_extract(self):
         self.assertEqual(files['main.css']['path'], os.path.join(settings.BASE_DIR, 'assets/bundles/main.css'))
         self.assertEqual(files['main.js']['path'], os.path.join(settings.BASE_DIR, 'assets/bundles/main.js'))
 
+    def test_default_ignore_config_ignores_map_files(self):
+        self.compile_bundles('webpack.config.sourcemaps.js')
+        chunks = get_loader('NO_IGNORE').get_bundle('main')
```

Comment:
Any reason for this `get_loader` to use a string the the other to be imported?

Reply:
It's just that the other is the default config and it's being used in multiple tests. Since this one is specific for this case, then there is no need for it to be a const.